scribbler/views.py

```diff
@@ -17,11 +17,11 @@
 from .utils import get_variables
 
 
-def build_scribble_context(scribble):
+def build_scribble_context(scribble, context={}):
     "Create context for rendering a scribble or scribble preview."
-    context = {
+    context.update({
```

Comment:
This is modifying a mutable default argument. This has the potential to leak state between pages/requests/templates.